README.md

```diff
@@ -33,60 +33,53 @@ developers, use the smaller
 
 ## Deploy the bundle
 
-```
 
+```
 juju deploy canonical-kubernetes
-
 ```
 
-This will deploy the Canonical Kubernetes offering with default constraints.
-This is useful for lab environments, however for real-world use you should
-provide high CPU and memory instances to kubernetes-worker.
+This will deploy the Canonical Distribution of Kubernetes offering with default
+constraints. This is useful for lab environments, however for real-world use
+you should provide higher CPU and memory instances to kubernetes-worker units.
 
-You can do this by editing the [bundle](https://github.com/juju-solutions/bundle-canonical-kubernetes)
-to fit your needs, it is commented for your convenience.
+You can increase the constraints by editing the
```

Comment:
Did we leave in commented constraints? I think that needs to be re-introduced. Its not a blocker, but we're currently telling a fib here....